check-headers.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+files=$(find . -name \*.go -type f -print0 | xargs -0 grep -L 'Licensed under the Apache License')
+if [ -n "$files" ]; then
+  echo "Missing license header in:"
+  echo "$files"
+  exit 1
+fi
```

Comment:
Nice, I'm going to steal this 😄

Reply:
It's not quite useful yet. It doesn't detect generated files, which won't have the license header, I need to add that

Reply:
@psiemens improved the script to ignore generated files